grpc.go

```diff
@@ -123,6 +167,27 @@ func onlineResources(resource onlineResource, nbResources int32) (uint32, error)
 	return count, nil
 }
 
+func offlineCPUResources(nbCpus uint32) error {
+	resource := onlineResource{
+		sysfsOnlinePath: sysfsCPUOnlinePath,
+		regexpPattern:   cpuRegexpPattern,
+	}
+
+	var count uint32
+	for i := uint32(0); i < onlineCPUMaxTries; i++ {
```

Comment:
This is slightly confusing - the function is *offlining* resources but you are iterating using an *online* variable. It might be worth renaming `onlineCPUMaxTries` to something like `cpuStateChangeMaxTries` to make it clearer?

Reply:
ok